Makefile

```diff
@@ -154,6 +157,7 @@ bump-kubevirtci:
 
 .PHONY: \
 	build-verify \
+	conformance \
```

Comment:
worth to keep order imo ?

Reply:
There is no order AFAIK

Reply:
you right, just good practice to order it imo, not need to be in this PR, 
just if you want, to put it before `clean` as the target it self

Reply:
I see no reason to try to keep an order in an unsorted list.